src/builder_state.rs

```diff
@@ -198,6 +198,9 @@ pub struct BuilderState<TYPES: NodeType> {
     /// constant fee that the builder will offer per byte of data sequenced
     pub base_fee: u64,
 
+    /// validated state
```

Comment:
Can you add a comment on why it has been introduced?

Reply:
Done!